src/project/internal/templatesrepository.cpp

```diff
@@ -107,6 +108,7 @@ Templates TemplatesRepository::readTemplates(const io::paths_t& files, const QSt
         Template templ;
         templ.categoryTitle = category;
         templ.meta = std::move(meta.val);
+        templ.isCustom = isCustom;
```

Comment:
It would be great to update this unit test: TEST_F(Project_TemplatesRepositoryTest, Templates)